### PR TITLE
Homework done!!

### PR DESCRIPTION
**Pull request by Salem8809**
### How difficult did you find this (0 - 10)? (0 very easy, 10 very difficult)
7

### How long did it take?
one hour

### Was there anything that you struggled with?  If so, what?
select the element

### Is there anything that you would like some further information on?
no

### Do you have any suggestions to improve this assignment?
no

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+planeteers[1]
+
+
+# Add "Heart" to the end of `planeteers`.
+
+
+planeteers.push("Heart")
+
+
+# Remove "Captain Planet" from the array (without using a method).
```
Comment (ghadeer-x) [inline app.rb:21]: to remove  "Captain Planet" from the array without using a method you can subtract it from the array
` planeteers  - ["Captain Planet"]`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+#### Bonus
+
+# Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+
+
+# Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+result = heroes.select {|word| word.include?"B"}
```
Comment (ghadeer-x) [inline app.rb:47]: Great way of solving this problem but using include will return any word that has "B" character in it , we want to check the first character so we can use

```rb
heroes.select{|heros| heros[0] === 'B'}
#or
heroes.select{|heros| heros.start_with? 'B'}
```

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+heroes.sort_by(&:downcase)
+
+# Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+heroes.shuffle
+
+#### Bonus
+
+# Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
```
Comment (ghadeer-x) [inline app.rb:41]: there is a cool method that does the magic for us it is `heroes.sample`

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+# Produce an array containing all of `ninja_turtle`'s keys using a method. [The Ruby documentation might help](http://ruby-doc.org/core-1.9.3/Hash.html).
+
+ninja_turtle.keys
+
+#### Bonus
+
+# Print out each key-value pair in the following format - "KEY's is equal to VALUE" -- using a method. [The Ruby documentation might help](http://ruby-doc.org/core-1.9.3/Hash.html).
+
+```rb
+# Your answer here
```
Comment (ghadeer-x) [inline app.rb:84]: we can use each to loop through the hash key then add the key and the value 
```rb
 ninja_turtle.each {|key , value| puts "#{key} is equal to #{value}"}
```

**Comment by ghadeer-x** [comment]
I can see you find some difficulties solving this hw no worries you will get better on time and it will take less time, Good Job 👍

**Comment by Salem8809** [comment]
> I can see you find some difficulties solving this hw no worries you will get better on time and it will take less time, Good Job 👍

Thank you Ghadeer for helping me out to find the better way to solve it

**Pull request closed without merging**